.gitignore

```diff
@@ -13,9 +12,13 @@ Sensors/*
 /pyvenv.cfg
 /bin/
 /lib/
+/lib64
```

Comment:
venv also creates this ~~folder~~ symlink to /lib on my machine (arch linux x86_64 , python 3.8)